superset-frontend/src/features/home/RightMenu.tsx

```diff
@@ -474,7 +475,7 @@ const RightMenu = ({
             <Menu.ItemGroup key="user-section" title={t('User')}>
               {navbarRight.user_profile_url && (
                 <Menu.Item key="profile">
-                  <Link to={navbarRight.user_profile_url}>{t('Profile')}</Link>
+                  <a href={navbarRight.user_profile_url}>{t('Profile')}</a>
```

Comment:
Should we preserve the native React component?

Reply:
Not sure how this happened, but nice catch, you Javascript enthusiast!